wiki/Glossary/en.md

```diff
@@ -742,18 +748,17 @@ Sliderbreak has two meanings:
 
```

Comment:
```
- In [osu!standard](/wiki/osu!standard):
  - Tapping the slider-start too early (this can cause the slider to shake).
  - Not collecting a slider tick (includes the hidden ones underneath the slider-start and slider-end).
  - Releasing the slider ball before it hits the slider-end.
    - This will not cause a miss, but instead will give the player a 100 with no combo increment.
- The case where the player does not obtain the max possible combo without missing an object.
```

Reply:
this is for lines 743 to 748
```
- In [osu!standard](/wiki/Game_Modes/osu!):
  - Tapping the slider-start too early (this can cause the slider to shake).
  - Not collecting a slider tick (includes the hidden ones underneath the slider-start and slider-end).
  - Releasing the slider ball before it hits the slider-end.
    - This will not cause a miss, but instead will give the player a 100 with no combo increment.
- The case where the player does not obtain the max possible combo without missing an object.
```